packages/flutter_test/lib/src/finders.dart

```diff
@@ -90,6 +90,25 @@ class CommonFinders {
   /// nodes that are [Offstage] or that are from inactive [Route]s.
   Finder byIcon(IconData icon, { bool skipOffstage: true }) => new _WidgetIconFinder(icon, skipOffstage: skipOffstage);
 
+  /// Looks for widgets that contain an [Icon] descendant displaying [IconData]
+  /// `icon` in it.
+  ///
+  /// Example:
+  ///
+  ///     // Suppose you have a button with icon 'arrow_forward' in it:
```

Comment:
Sample code should be in a `## Sample code` section, and should be marked up using triple-backtick blocks with a dart label, not indented, that way it'll get syntax-highlighted.

Reply:
Ah ok. Changed the rest of the docs in this file too.